encoders/nlp/UniversalSentenceEncoder/__init__.py

```diff
@@ -1,12 +1,67 @@
 __copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
+from typing import Optional
+
+import tensorflow_hub as hub
 import numpy as np
 
 from jina.executors.decorators import batching, as_ndarray
 from jina.executors.encoders.frameworks import BaseTFEncoder
 
 
+UNIVERSAL_SENTENCE_ENCODER = 'https://tfhub.dev/google/universal-sentence-encoder/4'
+MODEL_ENCODER_CMLM = "https://tfhub.dev/google/universal-sentence-encoder-cmlm/en-base/1"
```

Comment:
This is coming from my lack of knowledge, but there is only one possible combination of Model URL and preprocessor URL that deserve handling?

Reply:
would it make sense to have a boolean parameter (cmlm) or (masked model) to decide wether a preprocessing must be applied? or there exists only one single model with this API?

Reply:
Sorry, I know you asked this before I admit, I am not sure.. I tried to do it open for an extension if  in some near future we need to include this.

Reply:
https://tfhub.dev/google/collections/bert/1 <- In fact, it seems possible, we can include the feature now

Reply:
I can verify the paper, if it  is necessary.

Reply:
We can go with this for now I think. I think we just need to move the instantiatiom of private classes to post init, and no need to keep 'post_init` in the private classes